tests/framework/extensions/clusters/clusters.go

```diff
@@ -228,6 +229,7 @@ func NewRKE1ClusterConfig(clusterName string, client *rancher.Client, clustersCo
 					"stubDomains": "cluster.local",
 				},
 			},
+			EnableCRIDockerd: &falseBool,
```

Comment:
Should this be configurable? I remember saying something about this.

Reply:
addressed this by adding it to the config as a variable